common/changes/@microsoft.azure/openapi-validator-core/akhilailla-add_rpc_guideline_code_to_linter_command_2024-01-30-20-53.json

```diff
@@ -2,7 +2,7 @@
   "changes": [
     {
       "packageName": "@microsoft.azure/openapi-validator-core",
-      "comment": "Fix invalid self-dependency cycle bug",
+      "comment": "Upgrade version to 1.0.4",
```

Comment:
The idea here is to clarify what has changed. I.e. what are the changes coming in the new version.

Reply:
That is already added to the changelog 
When we do a version bump the individual changelogs gets deleted and get added to the main Changelog.json & changelog.md

This specific changelog is to address the version bump in this package